frontend/src/Utilities/Series/monitorOptions.js

```diff
@@ -6,6 +6,7 @@ const monitorOptions = [
   { key: 'pilot', value: 'Pilot Episode' },
   { key: 'firstSeason', value: 'Only First Season' },
   { key: 'latestSeason', value: 'Only Latest Season' },
+  { key: 'specials', value: 'Special Episodes'},
```

Comment:
`A space is required before '}'  object-curly-spacing`

Reply:
fixed with yarn lint-fix
Special Episodes renamed to Monitor Specials